src/vcpkg/archives.cpp

```diff
@@ -106,6 +106,7 @@ namespace
         static bool recursion_limiter_sevenzip = false;
         Checks::check_exit(VCPKG_LINE_INFO, !recursion_limiter_sevenzip);
         recursion_limiter_sevenzip = true;
```

Comment:
I think there is a lot more stuff like this you can delete now since the entire 7z dance is eaten by 7zr